src/ostorlab/runtimes/definitions.py

```diff
@@ -58,7 +58,7 @@ def container_image(self):
             return None
 
         tag = max(matching_tag_versions)
```

Comment:
Can you please rename to matching_version. I don't see the point of tag in there.